README.md

```diff
@@ -1,3 +1,125 @@
 # vald-client-java
 
 [![Maven Central](https://img.shields.io/maven-central/v/org.vdaas.vald/vald-client-java?style=flat-square)](https://search.maven.org/search?q=g:%22org.vdaas.vald%22%20AND%20a:%22vald-client-java%22)
+
+A Java gRPC client library for [Vald](https://github.com/vdaas/vald).
+
+## Usage
+
+### Dependency
+
+Add `vald-client-java` into your project dependency.
+
+For Maven `pom.xml`,
+
+```xml
+<dependency>
+    <groupId>org.vdaas.vald</groupId>
+    <artifactId>vald-client-java</artifactId>
+    <version>x.y.z</version>
+</dependency>
+<dependency>
+  <groupId>io.grpc</groupId>
+  <artifactId>grpc-netty-shaded</artifactId>
+  <version>x.y.z</version>
+</dependency>
+```
+
+Gradle `build.gradle`,
+
+```groovy
+implementation 'org.vdaas.vald:vald-client-java:x.y.z'
+implementation 'io.grpc:grpc-netty-shaded:x.y.z'
+```
+
+`io.grpc:grpc-netty-shaded` can be replaced by `io.grpc:grpc-okhttp` or `io.grpc:grpc-netty`.
+
+
+### Example
+
+```java
+package org.vdaas.vald.Example
+
+import io.grpc.ManagedChannelBuilder;
+
+import org.vdaas.vald.api.v1.vald.InsertGrpc;
+import org.vdaas.vald.api.v1.vald.SearchGrpc;
+import org.vdaas.vald.api.v1.vald.UpdateGrpc;
+import org.vdaas.vald.api.v1.vald.RemoveGrpc;
+
+import org.vdaas.vald.api.v1.payload.Insert;
+import org.vdaas.vald.api.v1.payload.Search;
+import org.vdaas.vald.api.v1.payload.Update;
+import org.vdaas.vald.api.v1.payload.Remove;
+import org.vdaas.vald.api.v1.payload.Object;
+
+public class Example {
+
+    String host = "gateway.vald.vdaas.org";
+    int port = 80;
+
+    public static void main(String[] args) throws Exception {
+
+        // Generate channel and stubs
+        ManagedChannel channel = ManagedChannelBuilder.forAddress(host, port).usePlaintext().build();
+        InsertGrpc.InsertBlockingStub istub = InsertGrpc.newBlockingStub(channel);
+        SearchGrpc.SearchBlockingStub sstub = SearchGrpc.newBlockingStub(channel);
+        UpdateGrpc.UpdateBlockingStub ustub = UpdateGrpc.newBlockingStub(channel);
+        RemoveGrpc.RemoveBlockingStub rstub = RemoveGrpc.newBlockingStub(channel);
+
+        // Insert
+        Object.Vector vec = Object.Vector.newBuilder()
```

Comment:
In java, I think it is better 😂
```suggestion
        ObjectVector vec = ObjectVector.newBuilder()
```

Reply:
thanks for commenting.
Object.Vector is a class generated by proto files, so we cannot modify the name of the class.